support/scripts/mkcpio

```diff
@@ -3,10 +3,37 @@ CPIOTOOL="$( which bsdcpio )"
 
 usage()
 {
-	echo "$0 [output.cpio] [CPIO root path]" 1>&2
+	echo "$0 [options] [output.cpio] [CPIO root path]" 1>&2
+	echo "-h	display this help screen" 1>&2
+	echo "-q	suppress non-critical output"
```

Comment:
I guess a 1>&2 is missing here

Reply:
The changes to this script were made to facilitate the grub build script. I required a quiet mode and I thought a compressed cpio image would be better. As for now, I have not figured out how to get a compressed cpio image to work with multiboot anyways, so the compression option is never used. For that reason, I think it would be fine to delay the compressed option until later. As for a a default quiet behavior, I didn't want to break the script for anyone that assumed a list of files would be outputted by default. I wasn't sure who used the script or how they used it, but if it doesn't break anything I think quiet by default would be preferred to maintain consistency with other Unix commands.